src/assets/i18n/fr-FR.json

```diff
@@ -76,7 +76,7 @@
       "sizeDesc": "Ajuste la taille du widget. Ceci est relatif à la taille globale.",
       "supportLink": "Soutien",
       "timezone": "Fuseau horaire",
-      "up": "UP",
+      "up": "Vers le haut",
```

Comment:
Google translate says this comes back to "to the top" which would be inaccurate as this string is used for moving something one up, not always to the top. Let me know if google translate is incorrect though as I rely on it and others to know :)

Reply:
The opposite of "vers le bas" is "vers le haut"
 
` "down": "VERS LE BAS", `

Indeed, `vers le haut `
can be understood "move up" or "move to top"

I've search where is use common.option.up  to have more context but I don't found.

I prefers rollback my change until we are sure  I've understood where   this word  would but use.

French is very complex language.

Reply:
Note : My english is really not perfect

Reply:
I'll trust you got it right, no worries! You've already corrected a bunch of others! IT will still get the point across :)